api/src/shop/stripe_constants.py

```diff
@@ -14,21 +13,88 @@
 STRIPE_SIGNING_SECRET = config.get("STRIPE_SIGNING_SECRET", log_value=False)
 
 
-class Type(str, Enum):
+def set_stripe_key(private: bool) -> None:
+    stripe.api_key = STRIPE_PRIVATE_KEY if private else STRIPE_PUBLIC_KEY
+
+
+set_stripe_key(True)
```

Comment:
Pass boolean argument by name

```suggestion
set_stripe_key(private=True)
```